jkwest-93/Smell-Ya-Later
# Jkw refactor app

#### What's this PR do?
- Relocates `App` to `components` directory
- Fleshes out `App.test`
- Adds new css styling to `App` and most other components
#### Where should the reviewer start?
- Check out `App.test`, `App`, and `App.css` for the bulk of this work, then look at the other files for the styling changes.
#### How should this be manually tested?
- Run `npm test` to check that tests are passing and then `npm start` to see app in the browser
#### Any background context you want to provide?
- After implementing `BrowserRouter`, I had to refactor my `App` tests
#### What are the relevant tickets?
- Close #12 
#### Screenshots (if appropriate)
- n/a
#### Questions:
- n/a
